jerry-core/parser/js/common.h

```diff
@@ -57,21 +57,21 @@ typedef enum
 
 /* Flags for status_flags. */
 
-/* Local identifier (var, function arg). */
+/** Local identifier (var, function arg). */
```

Comment:
Hm, would be better to convert these into an enum.

Reply:
I won't be pursuing that in this PR, which is solely about documentation fixes.